### DISCUSSION TITLE
Partially opt out of `dev` selections?

### DISCUSSION THREAD

**Discussion by dev-msp** [pkgxdev/pkgx#930] [Help] [answered]
Often I'll want `dev` to pick up things like `node`, `postgresql`, etc. 

But it also always picks up git when I'm in a git repo, and I never want it to do this - I have a git install already, and I prefer that pkgx doesn't control it.

Pardon if this is something I've missed in the docs, but is there a way to specify "dev, but not git/node/_"?

**Comment by jhheider** [comment]
this is certainly something that has come up before; it can be done manually, e.g. `env -git`. perhaps we need a `dependencies.ignore` section.

**Comment by mxcl** [comment, marked as answer]
`dev` is the least developed (lol) of our pieces. I have reserved time soonish. Certainly we understand the desire to have this  kind of control.
